Gopkg.toml

```diff
@@ -92,3 +102,6 @@
 [prune]
   go-tests = true
   unused-packages = true
+  [[prune.project]]
+    name = "k8s.io/code-generator"
+    unused-packages = false
```

Comment:
the code generation happens in scripts which are not pulled by dep if this is not specified. so had to set this to false.